lib/screens/v2/widget_instance/departures.ex

```diff
@@ -49,15 +49,20 @@ defmodule Screens.V2.WidgetInstance.Departures do
   end
 
   def serialize_section(%{type: :normal_section, rows: departures}, screen) do
-    rows = group_departures(departures)
+    rows =
+      group_departures(departures) ++
+        Enum.filter(departures, &match?(%{text: %Screens.Config.Dup.Override.FreeTextLine{}}, &1))
```

Comment:
Instead of splitting out notice rows and putting them at the end here, you can adjust the function passed to `Enum.chunk_by` in `group_departures` below.

Reply:
Great idea. Code is so much cleaner now.